### PR TITLE
Add GetLevel method

### PR DESCRIPTION
**Pull request by wangyuehong**
I came across https://github.com/rs/zerolog/issues/142 and it sounds like a good idea to expose getter method to return logger's level.

### CODE REVIEW

File: log.go [hunk 1]
```diff
@@ -251,6 +251,11 @@ func (l Logger) Level(lvl Level) Logger {
 	return l
 }
 
+// GetLevel returns the Level of logger
```
Comment (rs) [inline log.go]:
```suggestion
// GetLevel returns the current Level of l.
```

File: log_test.go [hunk 2]
```diff
@@ -453,6 +453,24 @@ func TestLevel(t *testing.T) {
...
+		WarnLevel,
+		ErrorLevel,
+		FatalLevel,
+		PanicLevel,
+		NoLevel,
+		Disabled,
+	}
+	for _, level := range levels {
+		if got, want := New(nil).Level(level).GetLevel(), level; got != want {
+			t.Errorf("GetLevel() got wrong value:\ngot:  %v\nwant: %v", got, want)
```
Comment (rs) [inline log_test.go]:
```suggestion
			t.Errorf("GetLevel() =  %v, want: %v", got, want)
```

**Comment by wangyuehong** [comment]
thanks for your time. words updated.

**Comment by j33ty** [comment]
@rs May I know when is the next master release? I need `GetLevel()`. (Thank you for the amazing work <3)

**Comment by rs** [comment]
New release done.

**Pull request merged**